app/scripts/lib/experiments/grouping-rules/signup-password-confirm.js

```diff
@@ -0,0 +1,33 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define((require, exports, module) => {
+  'use strict';
+
+  const BaseGroupingRule = require('./base');
+
+  module.exports = class SignupPasswordConfirmGroupingRule extends BaseGroupingRule {
+    constructor () {
+      super();
+      this.name = 'signupPasswordConfirm';
+    }
+
+    choose (subject = {}) {
+      const sampleRate = SignupPasswordConfirmGroupingRule.sampleRate(subject.env);
+
+      return !! subject.uniqueUserId && this.bernoulliTrial(sampleRate, subject.uniqueUserId);
```

Comment:
Two things will be helpful for testing and measuring success:

1. `control` and `treatment` groups. We can say 50% of users are in the experiment, of those 50%, half are in the `treatment` group, half in the `control` group. See [send-sms-install-link.js](https://github.com/mozilla/fxa-content-server/blob/9a8369ce6c8aa732d44fc5063be73c4c77050b95/app/scripts/lib/experiments/grouping-rules/send-sms-install-link.js#L38) for an example of how to do this.
2. The ability for testers to force themselves into the experiment and into a specific experiment group. We do this by specifying two query parameters, `forceExperiment` and `forceExperimentGroup`, e.g., `?forceExperiment=signupPasswordConfirm&forceExperimentGroup=treatment`. This gives us the ability to force the feature enabled for functional tests and for manual testing. [this section in send-sms-install-link.js](https://github.com/mozilla/fxa-content-server/blob/9a8369ce6c8aa732d44fc5063be73c4c77050b95/app/scripts/lib/experiments/grouping-rules/send-sms-install-link.js#L33) is the forceExperimentGroup half, `forceExperiment` comes beforehand.

In aggregate, I'd imagine something like this:

```js
choose (subject = {}) {
  if (! subject.uniqueUserId) {
     return;
  }

  const sampleRate = SignupPasswordConfirmGroupingRule.sampleRate(subject.env); 
  const isInExperiment = subject.forceExperiment === this.name ||
                this.bernoulliTrial(sampleRate, subject.uniqueUserId));

  if (isInExperiment) {
    const GROUPS = ['control', 'treatment'];
    return subject.forceExperimentGroup || this.uniformChoice(GROUPS, subject.uniqueUserId);
  }
}
```  

Which reminds me, I should put an example like this in the [recipe doc](https://github.com/mozilla/fxa-content-server/blob/9a8369ce6c8aa732d44fc5063be73c4c77050b95/app/scripts/lib/experiments/grouping-rules/README.md).